internal/services/network/express_route_circuit_peering_resource_test.go

```diff
@@ -223,6 +223,7 @@ resource "azurerm_express_route_circuit_peering" "test" {
   peer_asn                      = 100
   primary_peer_address_prefix   = "192.168.1.0/30"
   secondary_peer_address_prefix = "192.168.2.0/30"
+  ipv4_enabled                  = true
```

Comment:
if this is optional why are we adding it to all the tests?

Reply:
removde